util/formula_templater/formula.go

```diff
@@ -173,11 +173,27 @@ const caskTemplate = `cask "hashicorp-{{ .Product }}" do
   version "{{ .Version }}"
   sha256 "{{ .Architectures.DarwinAmd64SHA }}"
 
-  url "https://releases.hashicorp.com/{{ .Product }}/#{version}/{{ .Product }}_#{version}_darwin_amd64.dmg"
+  url "https://releases.hashicorp.com/{{ .Product }}/#{version}/{{ .Product }}_#{version}_darwin_amd64.dmg", 
+      verified: "hashicorp.com/{{ .Product }}/"
   name "{{ .Name }}"
   desc "{{ .Desc }}"
   homepage "{{ .Homepage }}"
 
+  livecheck do
+    url "https://github.com/hashicorp/{{ .Product }}"
+    strategy :git
+  end
+
   app "{{ .CaskApp }}"
+  pkg "{{ .CaskPkg }}"
+
+  uninstall script: {
```

Comment:
unsure if we want to customize this uninstall script option for _all_ casks?

Reply:
We should be able to add an if statement so that this is only added for vagrant's cask

Reply:
Added this in the latest commit!